changelog.dd

```diff
@@ -7,7 +7,8 @@ $(BUGSTITLE Library Changes,
 
 $(LI $(RELATIVE_LINK2 TypeInfo.initializer, `TypeInfo.init` has been renamed to
     `TypeInfo.initializer`.))
-)
+$(LI $(RELATIVE_LINK2 AA.clear, Associative array `clear` method to remove all
+    elements has been added.))
 )
```

Comment:
Wait, did you intend to remove the "2" from RELATIVE_LINK2 ? I compiled it with just RELATIVE_LINK and ddoc doesn't seem to know what that means.

Of course, it's impossible to actually build these freaking docs without lots of manual manipulation, so I have no idea if that's some externally defined macro...